### ISSUE TITLE
validation loss

### ISSUE THREAD

**Issue by hayvner** [Tramac/awesome-semantic-segmentation-pytorch#124]
The log only print train loss，I want to know validation loss，how to modify the file of train.py to print it？ 

新手小白求帮忙。  控制台只打印出了训练时候的loss值，我想看看验证时候的loss，怎样修改train.py 可以输出validation loss呢。
